modules/dcache/src/main/java/org/dcache/pinmanager/PinManager.java

```diff
@@ -196,6 +197,7 @@ public void isLeader()
         @Override
         public void notLeader()
         {
+            _log.info("Not assuming leader role. Probably there is another PinManager that already is entitled to the leader role.");
```

Comment:
The text here is a little misleading: the `notLeader` method is called when a pinmanager that was the leader is demoted so it is no longer the leader. Suggest something like:

    Dropping leader role. [...]

Also, it is usually (almost always) a bad idea to speculate in log messages: report what you know to be true; e.g.,

    Dropping leader role.  Cancelling expiration and unpin tasks.

Reply:
Thanks for clarifying this Paul!